### ISSUE TITLE
run yolov2 trained by myself

### ISSUE THREAD

**Issue by jackwei0117** [AlexeyAB/darknet#961]
Hi AlexeyAB,

Is it still possible to run the old yolov2(with it's .weight, .cfg, .data, .names files) I trained before yolov3 was proposed?
The format of .cfg file has been changed in v3, and I need to run yolov3 with the old yolov2 format.

This is the error message i got:
***
Error: l.outputs == params.inputs 
filters= in the [convolutional]-layer doesn't correspond to classes= or num= in [region]-layer 
***

This is the cfg file:
***
[net]
batch=64
subdivisions=16
height=608
width=608
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 2
exposure = 2
hue=.1

learning_rate=0.0001
max_batches = 45000
policy=steps
steps=100,5000,15000
scales=10,.1,.1

[convolutional]
batch_normalize=1
filters=32
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=64
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=128
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=256
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=3
stride=1
pad=1
activation=leaky

[maxpool]
size=2
stride=2

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=512
size=1
stride=1
pad=1
activation=leaky

[convolutional]
batch_normalize=1
filters=1024
size=3
stride=1
pad=1
activation=leaky

#######

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[route]
layers=-9

[reorg]
stride=2

[route]
layers=-1,-3

[convolutional]
batch_normalize=1
size=3
stride=1
pad=1
filters=1024
activation=leaky

[convolutional]
size=1
stride=1
pad=1
filters=1120
activation=linear

[region]
anchors = 1.08,1.19,  3.42,4.41,  6.63,11.38,  9.42,5.11,  16.62,10.52
bias_match=1
classes=1
coords=4
num=5
softmax=1
jitter=.2
rescore=1

object_scale=5
noobject_scale=1
class_scale=1
coord_scale=1

absolute=1
thresh = .8
random=1
***

Thank you

**Comment by AlexeyAB** [comment]
> Is it still possible to run the old yolov2(with it's .weight, .cfg, .data, .names files) I trained before yolov3 was proposed?

Yes, it is possible.


You should fix, or filters or classes:

`filters=1120` to 30

`classes=1` to 219

**Comment by jackwei0117** [comment]
Yeah, I found it as well.I tried to delete this issue before you answered,lol
